api_specifications/filestore-1.1.yaml

```
@@ -0,0 +1,387 @@
+openapi: 3.0.1
+info:
+  title: Filestore serivce APIs
+  description: |
+    APIs available for filestore service
+      -  Uploads different kinds of files to server including images and different document types.
+  contact:
+    name: eGovernments Foundation
+    email: contacts@egovernments.org
+  version: 2.0.0
+servers:
+- url: /filestore/v1
```

yamllint

[error] wrong indentation: expected 2 but found 0

(indentation)